### DISCUSSION TITLE
is there anyone can tell me why this 'Fatal error in launcher' appear？

### DISCUSSION THREAD

**Discussion by 1jiangxd** [pdm-project/pdm#2716] [Q&A]
after i installed pdm(```mamba install conda-forge::pdm```)，i tried 
```
pdm init
```
but it appears
```
(E:\aixuexi\anacondas\anacondabyj\p309wlda) E:\aixuexi\anacondas\anacondabyj\p309wlda>pdm init
Fatal error in launcher: Unable to create process using '"D:\bld\pdm_1708976248143\_h_env\python.exe"  "E:\aixuexi\anacondas\anacondabyj\p309wlda\Scripts\pdm.exe" init': ???????????
```
how can i fix this problem?is there anyone can tell me why this 'Fatal error in launcher' appear？

**Comment by pawamoy** [comment]
No idea :confused: It's definitely not a bug in PDM though, so I'll move this to a discussion.
I'd recommend asking for help in Anaconda's Q/A or Mamba's Q/A.

**Reply by 1jiangxd** [reply]
thank you for your help，
then i tried this to install：
```
(Invoke-WebRequest -Uri https://pdm-project.org/install-pdm.py -UseBasicParsing).Content | python -
```
it shows `installation successful`，then i tried
```
pdm init
```
but it still shows 😕  😕  😕 ：
```
Fatal error in launcher: Unable to create process using '"D:\bld\pdm_1708976248143\_h_env\python.exe"  "E:\aixuexi\anacondas\anacondabyj\p309wlda\Scripts\pdm.exe" init': ???????????
```
